support-workers/src/test/scala/com/gu/zuora/ZuoraITSpec.scala

```diff
@@ -21,15 +21,15 @@ import scala.concurrent.duration._
 @IntegrationTest
 class ZuoraITSpec extends AsyncFlatSpec with Matchers {
 
-  def uatService: ZuoraService =
+  def codeService: ZuoraService =
     new ZuoraService(
-      Configuration.load().zuoraConfigProvider.get(true),
```

Comment:
What’s the reason for making this no longer a test user?

Reply:
It doesn't make any difference either way because test users and non-test users will use the SANDBOX environment in these tests so I thought it was less confusing to new comers to have to think about what that `true` meant.